### PR TITLE
Fix overlapping title and clientShares in Planner Client Weights

### PR DESCRIPTION
**Pull request by AnttiVaarala**

### CODE REVIEW

File: frontend/src/pages/PlannerWeightsPage.module.scss [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
+    align-items: flex-start;
+    margin-right: 25px;
   }
 
   .clientShares {
     display: flex;
-    flex-grow: 1;
-    justify-content: flex-end;
-    margin-right: 25px;
+    margin: 0 34px 0 auto;
```
Comment (IidaKainu) [inline frontend/src/pages/PlannerWeightsPage.module.scss:17]: Now that `margin-right: 25px` is added to the .title, this has some unnecessary margin
```suggestion
    margin-left: auto;
```

File: frontend/src/pages/PlannerWeightsPage.module.scss [hunk 2]
```diff
@@ -2,20 +2,19 @@
...
   @extend .layout-row;
-  margin-left: 25px;
+  margin: 0 25px;
   align-items: flex-start;
 
   .title {
     @extend .layout-row;
     gap: 12px;
-    align-items: center;
+    align-items: flex-start;
```
Comment (IidaKainu) [inline frontend/src/pages/PlannerWeightsPage.module.scss:11]: Noticed we can get rid of the extra div around the infoTooltip by setting the align as baseline
```suggestion
    align-items: baseline;
```

**Review by IidaKainu — CHANGES_REQUESTED** [review]
Just a few small things

File: frontend/src/pages/PlannerWeightsPage.module.scss (hunk 1 above)
Comment (AnttiVaarala) [reply frontend/src/pages/PlannerWeightsPage.module.scss:17]: If there's not margin on the right, element overflows and part of it is hidden, the 34px is (padding 16px + border 1px)*2

Comment (IidaKainu) [reply frontend/src/pages/PlannerWeightsPage.module.scss:17]: I can't seem to replicate the overflowing, would you mind showing how to? (on slack or here)

**Review by IidaKainu — APPROVED** [review]

**Review by IidaKainu — APPROVED** [review]

**Pull request merged**